packages/ui/src/utils/camel-uri-helper.test.ts

```diff
@@ -0,0 +1,77 @@
+import { CamelUriHelper } from './camel-uri-helper';
+
+describe('CamelUriHelper', () => {
+  describe('getUriString', () => {
+    it.each([
+      [undefined, undefined],
+      [null, undefined],
+      [88, undefined],
+      [true, undefined],
+      [false, undefined],
+      ['', undefined],
+      [{ uri: '' }, undefined],
+      [{ uri: undefined }, undefined],
+      [{ uri: null }, undefined],
+      [{ uri: 88 }, undefined],
+      [{ uri: {} }, undefined],
+      ['a string', 'a string'],
+      [{ uri: 'a string' }, 'a string'],
+    ])('should return `%s` for `%s`', (value, expected) => {
+      expect(CamelUriHelper.getUriString(value)).toBe(expected);
+    });
+  });
+
+  describe('uriSyntaxToParameters', () => {
+    it.each([
```

Comment:
@apupier the following test is defined using 3 keys, `syntax`, `uri`, and `result`.

We can read it as for a given `uri`, using a `syntax`, we get this object in `result`.